keel/leads/serializers.py

```diff
@@ -0,0 +1,17 @@
+from rest_framework import serializers
+from rest_framework import validators
+from .models import CustomerLead
+
+
+class CustomerLeadSerializer(serializers.ModelSerializer):
+
+    email = serializers.EmailField(required=True, 
+                                    validators=[validators.UniqueValidator(queryset=CustomerLead.objects.all())])
+    phone_number = serializers.IntegerField()
+    lead_source = serializers.ChoiceField(choices=CustomerLead.LEAD_SOURCE)
+    resolution = serializers.ChoiceField(choices=CustomerLead.RESOLUTION)
```

Comment:
we don't need resolution from API request...that would be handled on our side

Reply:
updated please check